Library/Homebrew/dev-cmd/bump.rb

```diff
@@ -38,23 +40,44 @@ def bump
       raise UsageError, "`--limit` must be used with either `--formula` or `--cask`."
     end
 
-    formulae_and_casks = if args.formula?
-      args.named.to_formulae.presence
-    elsif args.cask?
-      args.named.to_casks.presence
-    else
-      args.named.to_formulae_and_casks.presence
-    end
+    formulae_and_casks =
+      if args.formula?
+        args.named.to_formulae
+      elsif args.cask?
+        args.named.to_casks
+      else
+        args.named.to_formulae_and_casks
+      end&.sort_by do |formula_or_cask|
```

Comment:
`end&` is a bit weird; I'd split this out into a variable.

Reply:
I'd actually based this part of the code on [`dev-cmd/livecheck.rb`](https://github.com/Homebrew/brew/blob/5fed5536f4cf35aa86e300cb72fc5c4194b17587/Library/Homebrew/dev-cmd/livecheck.rb#L99), so should that be changed too?

Reply:
Ideally, yeh!